File: src/main/java/de/lukaseichler/recurrentneuralnetwork/structure/SmoothingLayer.java
package de.lukaseichler.recurrentneuralnetwork.structure;

import de.lukaseichler.recurrentneuralnetwork.activation.TanhActivation;

/**
 * @author leichler
 */
public class SmoothingLayer extends Layer {

    public SmoothingLayer() {
        this(0);
    }

    public SmoothingLayer(int count) {
        super(count, new TanhActivation());
    }

}

File: src/main/scala/de/lukaseichler/recurrentneuralnetwork/structure/SmoothingLayer.scala
package de.lukaseichler.recurrentneuralnetwork.structure

import de.lukaseichler.recurrentneuralnetwork.activation.TanhActivation

/**
 * @author Lukas Eichler
 */
class SmoothingLayer(count: Int = 0) extends Layer(count, new TanhActivation) {
  def this() = this(0)
}
